test/jdk/sun/security/provider/PolicyParser/ExtDirs.java

```diff
@@ -26,16 +26,15 @@
  * @bug 4215035
  * @summary standard extensions path is hard-coded in default
  *      system policy file
- * @run main/manual ExtDirs
+ * @run main ExtDirs
  */
 
 /*
- * Run this test manually with:
- * java    -Djava.security.manager \
- *         -Djava.ext.dirs=./ExtDirsA:./ExtDirsB \
- *         -Djava.security.policy==./ExtDirs.policy \
- *         -Djava.security.debug=parser \
- *         ExtDirs
+ * @test
```

Comment:
Why are these duplicated here?

Reply:
Multiple "run" in a single test fails the whole test when one test run fails and all subsequent test run get terminated. Multiple tests with only one "run" each allows all test run to be excused, and they can be run in parallel.

Reply:
I have not seen this used in any other tests. It is a good learning, I didn't knew that multiple tests in single file are run in parallel. It seems like good option. Do you know of any other regression test written this way?

Reply:
I didn't know that until Leonid posted this [comment](https://github.com/openjdk/jdk/pull/9813#discussion_r942908285) in one of my PRs. We do have a lot of tests written in this way and each test run will be given a unique id after the file name. For example, ExtDirs.java has two test run, and they will be named as ExtDirs.java#id0 and ExtDirs.java#id1. We can search "name:*.java#id*" in a Core Lib Mach5 run, it will show thousands of results.

Reply:
Thanks for this. Fix looks good to me.